src/permitInfo/fetchPermitInfo.ts

```diff
@@ -100,37 +116,48 @@ async function fetchPermitInfo(
   }
 }
 
+// Fn can only be called 2x/second
+const throttle = pThrottle({
+  limit: 2,
+  interval: 1000,
+})
+
+const throttledGetTokenPermitInfo = throttle(getTokenPermitInfo)
```

Comment:
Throttling the call to try and avoid RPC failures.
Still, every time I run the recheck, new tokens are identified as permittable.

Reply:
Does it fail a lot? I mean, how many calls w e do per token? shouldn't break if we use our RPC (nodereal)
Is not 2 per second too slow?  
What do you mean with "Still, every time I run the recheck, new tokens are identified as permittable." is the script never finishing?

Reply:
Ran it today again to get an example:

![image](https://github.com/cowprotocol/token-lists/assets/43217/d992e130-7028-47dd-aa04-d31fdf80b47b)

Will see how I can mark those failures as retryable